fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/dao/impl/ResourceDAOImpl.java

```diff
@@ -549,6 +564,21 @@ public Resource insert(Resource resource, List<ExtractedParameterValue> paramete
 
             resource.setId(stmt.getLong(7));
 
+            if (large) {
+                // Use the long id to update the record in the database.
+                String largeStmtString = String.format(LARGE_BLOB, getSchemaName(), resource.getResourceType());
```

Comment:
For table access, don't need to qualify with the schema name, although it doesn't hurt.

Reply:
Thanks